### ISSUE TITLE
Is it possible to merge 2 projects?

### ISSUE THREAD

**Issue by herosnowman** [alicevision/meshroom#473]
Thanks for stopping by my problem. Basically I have 2 project and both have calculated to the end of texturing node. Both scanned objects are next to each other(in real world). Is it possible to make 1 mesh from those 2 projects, inside of meshroom?
Any help appreciated!

**Comment by natowi** [comment]
This depends on the overlap. Have a look at [Meshlab ](http://www.meshlab.net/#download)([demo](https://vimeo.com/26362121)).

**Comment by fabiencastan** [comment]
Hi,

We don't have a node to merge 2 meshes (that would be a great and easy to do contribution if someone is looking to make a first simple contribution).

If you have common cameras, you can use the `SfMTransform` to align one SfM onto the other (but it will apply the transform to the SfM and you will not be able to apply it on the Mesh, so the only solution would be to recompute the second one).

So in conclusion, that would be great to have but it's yet there, so as suggested by @natowi, the easiest way is to do it in another software.

Best,
